src/test/boring-test.js

```diff
@@ -1,6 +1,6 @@
 var assert = require('assert');
 
-describe('Boring entrypoint', function() {
+describe.only('Boring entrypoint', function() {
```

Comment:
the fuck is this

Reply:
It's fixed... this only runs THIS unit tests.